Source/Pawnmorphs/Esoteria/Hybrids/RaceShiftUtilities.cs

```diff
@@ -313,10 +313,7 @@ private static void ValidateReversion(Pawn pawn)
             }
 
             if (graphicsComp == null)
-            {
-                Log.Error($"trying to validate the graphics of {pawn.Name} but they don't have an {nameof(InitialGraphicsComp)}!");
-
-            }
+                return;
```

Comment:
there should still be at least a warning here if this happens as this could cause issues with certain aliens or explicit morphs.